base/compiler/ssair/ir.jl

```diff
@@ -746,9 +726,7 @@ end
 
 function count_added_node!(compact::IncrementalCompact, @nospecialize(v))
     needs_late_fixup = false
-    if isa(v, SSAValue)
-        compact.used_ssas[v.id] += 1
-    elseif isa(v, NewSSAValue)
+    if isa(v, NewSSAValue)
```

Comment:
Presumably `NewSSAValue` should be treated the same in userefs as a regular SSAValue?